Commands/guessgame.py

```
@@ -0,0 +1,190 @@
+    hint = ascii.first_frame(self, channel, emote_url)
+    self.send_privmsg(channel, hint)
+
+
+def reveal_emote(self, channel, emote):
+    self.send_privmsg(
+        channel, f"The emote was {emote} Disappointing performance :Z")
+
+    if self.currentRound + 1 == self.numRounds:
+        # End the game if all rounds are done
+        self.send_privmsg(
+            channel, f"Game has ended.")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)